sale_stock_return_request/models/stock_return_request.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2019 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import api, fields, models
+from odoo.osv import expression
+
+
+class StockReturnRequest(models.Model):
+    _inherit = "stock.return.request"
+
+    filter_sale_order_ids = fields.Many2many(
+        comodel_name="sale.order",
+        relation="stock_return_request_so_filter",
+        string="Selected order(s)",
+        copy=False,
+        domain=[("state", "in", ["sale", "done"])],
```

Comment:
```suggestion
        domain=[("state", "=", "sale")],
```
The donated state no longer exists.

Reply:
Thanks